animeal/src/Flows/Main/Modules/More/ViewModel/Mappers/MoreViewModel.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+final class MoreViewModel: MoreViewModelLifeCycle, MoreViewInteraction, MoreViewState {
+
+    // MARK: - Dependencies
+    private let model: MoreModelProtocol
+    private let coordinator: MoreCoordinatable
+    private let mapper: MoreItemViewMappable
+
+    // MARK: - State
+    var onActionsHaveBeenPrepared: (([MoreItemView]) -> Void)?
+
+    // MARK: - Initialization
+    init(
+        coordinator: MoreCoordinatable,
+        mapper: MoreItemViewMappable = MoreItemViewMapper(),
+        model: MoreModelProtocol
+    ) {
+        self.coordinator = coordinator
+        self.mapper = mapper
+        self.model = model
+    }
+
+    // MARK: - Life cycle
+    func load() {
+        let actions = model.fetchActions()
+        onActionsHaveBeenPrepared?(
+            actions.map {
+                mapper.mapActionModel($0)
```

Comment:
retain cycle here?

Reply:
as discussed

Reply:
yep, possible enhancement (to review faster and simpler): 
let actions = model.fetchActions().map { mapper.mapActionModel($0) }
onActionsHaveBeenPrepared?(actions)